### DISCUSSION TITLE
Telling last time an account was reconciled

### DISCUSSION THREAD

**Discussion by BenjaminMichaelis** [firefly-iii/firefly-iii#9300] [Firefly III]
Hi! Love the app btw, and appreciate all the effort you put in it.

I was curious; when I look at an asset account, I can't quickly tell if it is reconciled without going to it's transactions and seeing when the last transaction with the ✔️ is. I was curious if there is a way to at a glance from like the assets page to be able to see the last "reconciled" date or similar, as when I look at my asset page I want to quickly be able to see what accounts have I reconciled this month and which have I forgotten.

Also maybe I just don't know how it works properly yet, but is there a way to filter transactions down to non-reconciled ones? I notice sometimes I let a non-reconciled transaction through the cracks and I would love a quick way to filter down to see these so I can find old ones. Best solution I have so far is using the API, but just wanted to check.

Thank you!

**Comment by JC5** [comment]
Mmm, the time of the last reconciliation isn't available yet, but you can use `reconciled:true` (or false) to search for reconciled transactions. Just enter `reconciled:true` in the search (perhaps combined with `account_id:4` and you should be good to go.
